src/assimp/assimp.cpp

```diff
@@ -296,6 +296,26 @@ SamplerData SceneConverter::convertTexture(const aiMaterial& material, aiTexture
             }
         }
 
+        bool sRGBTextures = false;
+        options->getValue("sRGBTextures", sRGBTextures);
```

Comment:
In incluce/vsg/core/Value.h there is a convenience template function vsg::value<>(..), this is used in several places in assimp.cpp to streamline the getting of values from objects with support for using a default when it's not available.